starter_code/express-spotify-app/views/artists.ejs

```diff
@@ -0,0 +1,8 @@
+<body>
+    <div class="container">
+
+    </div>
+    </div>
+</body>
+
+</html>
```

Comment:
If you are using a main-layout, you don't need the body tags, neither the html one.